lib/fathead/node_js/parse.js

```diff
@@ -33,20 +33,19 @@ function getAbstract(obj) {
     var a = obj.desc ? '<p>' + obj.desc + '</p>' : '',
         code = '',
         stability = '';
+    // strip <a> and <p> HTML tags but keep others.
+    a = a.replace(/<\s*p*>/ig, '');
```

Comment:
@willyb321 we need to move this above so it operates on `obj.desc` -- we actually want the `<p>` tags that we're putting into the abstract a few lines up. We just don't want any `<a>` tags.

We need the `<p>` so that the description text will be properly padded and separated from the `<pre><code>`:

```html
<section>
    <p>Description text with a <code>example</code> and text</p>
    <pre><code>var foo = "bar"</code></pre>
    <p>possibly more text</p>
</section>
```